rust/worker/src/blockstore/arrow/blockfile.rs

```diff
@@ -373,6 +523,81 @@ mod tests {
         }
     }
 
+    #[parameterized(operation = {"gt", "gte", "lt", "lte"})]
+    #[parameterized_macro(tokio::test)]
+    async fn test_gt(operation: &str) {
+        let tmp_dir = tempfile::tempdir().unwrap();
+        let storage = Storage::Local(LocalStorage::new(tmp_dir.path().to_str().unwrap()));
+        let blockfile_provider = ArrowBlockfileProvider::new(storage);
+        let writer = blockfile_provider.create::<&str, u32>().unwrap();
+        let id = writer.id();
+
+        // Each block is 16384 bytes. A key is ~4 bytes and value is ~4 bytes. So total
```

Comment:
comment assumes fixed block size. we will change the block size to be configurable and also will default to 16MB